src/ansys/fluent/core/filereader/case_file.py

```diff
@@ -304,7 +304,7 @@ def __init__(
                     dirname(project_filepath),
                     Path(project_filepath).name.split(".")[0] + ".cffdb",
                 )
-                case_filepath = Path(
+                settings_filepath = Path(
```

Comment:
regarding the call to _get_case_filepath_from_flprj, that's literally just getting the Case. So for Settings file, we should leave this out.